explorer/src/lib.rs

```diff
@@ -108,6 +107,7 @@ impl hyperware_app_framework::State for State {
 
 impl State {
     pub fn make_filter(hypermap: &hypermap::Hypermap) -> eth::Filter {
+        println!("hypermap.address {}", &hypermap.address().to_string());
```

Comment:
Change this from `println!()` to `print_to_terminal()` with a verbosity of `2` so it only prints to terminal if the user has a higher verbosity mode enabled.